print/components/PicassoChunk.vue

```diff
@@ -138,6 +243,14 @@ export default {
   transform: rotate(-90deg);
   position: relative;
   top: 320px;
+
+  .fullwidth {
+    width: 960px;
```

Comment:
Print preview looks fine, but it breaks in the printed pdf (maybe due to the ecamp3 header and page number footer). Maybe try to reduce this width a little. Soon we should be able to remove the rotate 90deg hack, when the browserless chrome supports switching to landscape mode.